.github/workflows/ci.yml

```
@@ -5,25 +5,25 @@ on:
       - master
       - release-*
   pull_request:
-  pull_request_target:
   workflow_dispatch:
 env:
   DOTNET_NOLOGO: true
 jobs:
   build:
-    if:
-      (github.event_name == 'pull_request_target' && github.event.pull_request.user.login == 'dependabot[bot]') ||
-      (github.event_name == 'pull_request' && github.event.pull_request.user.login != 'dependabot[bot]') ||
-      github.event_name == 'push' || github.event_name == 'workflow_dispatch'
-    name: ${{ matrix.name }}
+    name: ${{ matrix.os-name }}-${{ matrix.framework }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os: [ windows-2019, ubuntu-20.04 ]
+        framework: [ net6.0, net472 ]
```

Almost missed this - test projects still have netcoreapp3.1 as well
```suggestion
        framework: [ netcoreapp3.1, net6.0, net472 ]
```